.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,9 +2,9 @@
 
 ## Checklist
 
-* [ ] Tests written for new code (and old code if feasible)
-* [ ] Linter and other CI checks pass
-* [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md))
+-   [ ] Tests written for new code (and old code if feasible)
```

Comment:
Not sure of the value of applying prettier to markdown files. Fine by me if we do, but also fine if we don't.